sys/mips/mips/trap.c

```diff
@@ -1160,8 +1160,13 @@ trap(struct trapframe *trapframe)
 		msg = "USER_CHERI_EXCEPTION";
 		fetch_bad_instr(trapframe);
 		log_c2e_exception(msg, trapframe, type);
-		i = SIGPROT;
-		ucode = cheri_capcause_to_sicode(trapframe->capcause);
+		if (trapframe->capcause == CHERI_EXCCODE_TLBSTORE) {
+			i = SIGSEGV;
+			ucode = SEGV_STORETAG;
```

Comment:
So MIPS will also raise SEGV now? I guess that means the cheritest can unconditionally check for SEGV?

Reply:
Yes. See the first hunk of the diff, which does so for cheritest.c.

Reply:
I misread the
```
#ifdef __riscv
	  .ct_si_trapno = EXCP_STORE_AMO_CAP_PAGE_FAULT,
#else
	  .ct_si_trapno = TRAPNO_CHERI,
#endif
```
as being the signal number.